### PR TITLE
QR-Code -> QR code

### PR DESCRIPTION
**Pull request by yegortimoshenko**
This commit fixes spelling of "QR code".

### CODE REVIEW

File: app/src/main/res/values-de-rDE/strings_main.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <!-- Buttons -->
     <string name="button_cancel">Abbrechen</string>
     <string name="button_enter_details">Details eingeben</string>
-    <string name="button_scan_qr">QR-Code scannen</string>
+    <string name="button_scan_qr">QR code scannen</string>
```
Comment (joschi) [inline app/src/main/res/values-de-rDE/strings_main.xml]: "QR-Code" is the correct spelling in German.

File: app/src/main/res/values-fr-rFR/strings_main.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <!-- Buttons -->
   <string name="button_cancel">Annuler</string>
   <string name="button_enter_details">Ajouter les détails</string>
-  <string name="button_scan_qr">Scanner un QR-Code</string>
+  <string name="button_scan_qr">Scanner un QR code</string>
```
Comment (joschi) [inline app/src/main/res/values-fr-rFR/strings_main.xml]: Correct spelling would be "Code QR".
https://fr.wikipedia.org/wiki/Code_QR

File: app/src/main/res/values-it-rIT/strings_main.xml [hunk 3]
```diff
@@ -69,6 +69,6 @@
         <b>Qualsiasi voce aggiunta verrà persa</b>\n\nPer continuare ad utilizzare andOTP, vai nelle
         <b>Impostazioni</b> e imposta la <b>Crittografia del database</b> su <b>Password / PIN</b>.</string>
     <!-- Shortcuts -->
-    <string name="shortcut_name_scan_qr">Scansione QR-Code</string>
+    <string name="shortcut_name_scan_qr">Scansione QR code</string>
```
Comment (joschi) [inline app/src/main/res/values-it-rIT/strings_main.xml]: https://it.wikipedia.org/wiki/Codice_QR

File: app/src/main/res/values-ca-rES/strings_main.xml [hunk 4]
```diff
@@ -69,6 +69,6 @@
         <b>Any entries that are added will get lost.</b>\n\nTo still be able to use andOTP you can go
         to the <b>Settings</b> and switch the <b>Database encryption</b> to <b>Password / PIN</b>.</string>
   <!-- Shortcuts -->
-  <string name="shortcut_name_scan_qr">Scan QR-Code</string>
+  <string name="shortcut_name_scan_qr">Scan QR code</string>
```
Comment (joschi) [inline app/src/main/res/values-ca-rES/strings_main.xml]: https://es.wikipedia.org/wiki/C%C3%B3digo_QR

**Comment by yegortimoshenko** [comment]
@joschi: thank you so much! Applied. Let me know if you find something like this in more locales.

**Comment by flocke** [comment]
You didn't really have to concern yourself with the locales. The original (English) files will be uploaded to Crowdin before the next release and the translations will overwrite the changes you made. But I will merge it for now.

**Pull request merged**